readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
Handy list, thanks,

"Functional properties"  Do we need to use function that returns a property there? Or it can be its static result? Your example doesn't look like its a function, more like its a simple interpolation.